go/apis/paddlepaddle/v1alpha1/types.go

```diff
@@ -36,7 +36,10 @@ type TrainingJob struct {
 // TrainingJobSpec is the spec for a TrainingJob resource
 type TrainingJobSpec struct {
 	// General job attributes.
-	Image             string              `json:"image,omitempty"`
+	Image string `json:"image,omitempty"`
+	// If you want to use the hostnetwork instead of container network
+	// portmanager is necessary.
```

Comment:
Do we have a link to introduce the **portmanager**? The user may not know what's a **portmanager** is.

Reply:
@Yancey1989 ok, let me add a link